software_training_assignment/src/resetPos.cpp

```diff
@@ -0,0 +1,31 @@
+#include <std_srvs/Empty.h>
+#include "resetPos.h"
+#include "killTurtles.h"
+#include "spawnTurtles.h"
+
+bool resetPos(std_srvs::Empty::Request &req,
+	      std_srvs::Empty::Response &res)
+{
+	ros::NodeHandle n;
```

Comment:
should pass the NodeHandle by reference, you ideally should only need 1 NodeHandle per node you write